src/main.rs

```diff
@@ -34,38 +38,7 @@ use crate::aquariwm::AquariWm;
 /// The main entrypoint for AquariWM; `main` is responsible for initalization/setup.
 fn main() -> xcb::Result<()> {
 	// Initialize the default [`tracing`] subscriber so that all logged messages are printed to
-	// the console. AquariWM values the heavy use of log messages throughout the codebase. You
-	// should use the different log levels as follows:
-	//
-	// - `Trace`
-	//   You should use the `Trace` level often - it is the most verbose log level, and should
-	//   be used for any small operation which could affect the running of the window manager.
-	//   For example, you should generally make a `Trace` level message for every request sent to
-	//   the X server.
-	//
-	// - `Debug`
-	//   Use the `Debug` level for a more easily understood grouping of operations. For example,
-	//   a `Debug` level message might say:
-	//   > Registering for events on existing windows
-	//
-	// - `Info`
-	//   The `Info` log level should be used relatively sparingly. It describes bigger steps in
-	//   the operation of AquariWM, for example:
-	//   > Initializing AquariWM
-	//   or:
-	//   > Loading configuration
-	//
-	// - `Warn`
-	//   The `Warn` level should be used when actual functionality or features of the window
-	//   manager are hindered or restricted because of a problem encountered, but that problem was
-	//   not critical to the continued running of the window manager. It can also be used for
-	//   alerting users of unwise or not-recommended configuration setups, even if they may wish
-	//   to proceed with them nonetheless. That might particularly include the use of experimental
-	//   features, for example.
-	//
-	// - `Error`
-	//   Use the `Error` log level only for critical errors that can cause entire components or
-	//   the window manager itself to no longer be able to continue to run properly.
```

Comment:
Not yet, but I am planning on writing it in CONTRIBUTING.md or somewhere like that - this just isn't the appropriate location for that information.

Reply:
It might be good to have in doc comments perhaps? Whether that would be crate level documentation or something, I'm not sure.  I don't know if that is appropriate or not, but as this is a binary crate, my thinking was that people reading _these_ docs are contributors.  It does make sense to me to put it in CONTRIBUTING.md, at least in addition, however - CONTRIBUTING.md is for contributing guidelines, which these seem to be.

Reply:
I believe that information could also be useful to a general user looking to debug something, so in my opinion CONTRIBUTING.md isn't the ideal location. Maybe a page in the component documentation about logging that would be linked to in CONTRIBUTING and in some eventual Troubleshooting section could be better.

Reply:
> I believe that information could also be useful to a general user looking to debug something, so in my opinion CONTRIBUTING.md isn't the ideal location. Maybe a page in the component documentation about logging that would be linked to in CONTRIBUTING and in some eventual Troubleshooting section could be better.

I see, that makes sense. I'd say in that case it should be 'duplicated': two descriptions, one more appropriate for contributors, the other for users troubleshooting?